### DISCUSSION TITLE
Conflicting with usami-k/search-history plugin

### DISCUSSION THREAD

**Discussion by furopi** [PatrickF1/fzf.fish#297] [Ideas]
usami-k/search-history should also be added to [Uninstalling other fzf plugins](https://github.com/PatrickF1/fzf.fish/wiki/Uninstalling-other-fzf-plugins), imho, as it's conflicting with fzf.fish... 
Took me quite a while to figure that out.

Cheers,
Florian

**Comment by PatrickF1** [comment]
sure, thanks
